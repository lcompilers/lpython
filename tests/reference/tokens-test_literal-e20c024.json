{
    "basename": "tokens-test_literal-e20c024",
    "cmd": "lpython --no-color --show-tokens {infile} -o {outfile}",
    "infile": "tests/errors/test_literal.py",
    "infile_hash": "ac9e219faa40c6554983087e8ac198c323f2e5af284b8689f5608f76",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "tokens-test_literal-e20c024.stdout",
    "stdout_hash": "5eef739e7cd2a03fa9b24992f8c4436186d69d8f3a045a00e5a0d6a1",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}