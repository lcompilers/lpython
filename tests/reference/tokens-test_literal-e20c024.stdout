(KEYWORD "def") 0:2
(TOKEN "identifier" test_literal1) 4:16
(TOKEN "(") 17:17
(TOKEN ")") 18:18
(TOKEN ":") 19:19
(NEWLINE) 20:20
(TOKEN "indent") 21:24
(TOKEN "identifier" x) 25:25
(TOKEN ":") 26:26
(TOKEN "identifier" i32) 28:30
(TOKEN "=") 32:32
(TOKEN "integer" 123) 34:37
(EOF) 38:38
